help/en/html/tools/scripting/Start.shtml

```diff
@@ -280,10 +280,26 @@ You can also use the symbols "True" and "False" respectively.
       send a DCC packet to the rails you type:</p>
       <pre>
    
-   dcc.sendPacket([0x01, 0x03, 0xbb], 4) 
+   <code>dcc.sendPacket([0x01, 0x03, 0xbb], 4) </code>
 </pre>This sends that three-byte packet four times, and then
 returns to the command line.
 
+
+    <p>If the returned object is an ID Tag or RailCom Object, the return value 
+    ( from Java #toString() ) is the Hardware Address.<br>
```

Comment:
RailCom objects ARE IDTags, so there is no need to specify them separately.

Reply:
IDTags are named beans.  The toString method they inherit from abstract name bean provides the system address.